### PR TITLE
Create Understanding Cost Estimates in Control Portal 

### PR DESCRIPTION
**Pull request by gavin-lai**
Digital Team: Please DO NOT merge until we conduct an internal feedback gather and have approval from other teams.  
Create  Understanding Cost Estimates in Control Portal KB, seeking for feedback from teams

### CODE REVIEW

**Comment by gavin-lai** [comment]
Digital team: It is good to go for merging, thank you

File: Control Portal/understang-cost estimates in control.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+The estimate for a server will show:
+ - All infrastructure costs (CPU, Memory, disk) and OS licenses (where applicable) from the data center
+ - Managed Services costs (Managed OS, Active Directory, Databases, Web server)
+
+### Network
+
+Under Network menu, the cost of VLANs, Share Load Balanacer, DNS and Site to Site VPN can be found within the configuration page of the service.
+
+For Networks:
```
Comment (chrislittle) [inline Control Portal/understang-cost estimates in control.md]: maybe on these sub areas add a ##### (5 # title)

File: Control Portal/understang-cost estimates in control.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+DNS:
+
+![DNS](../images/estimator/dns.png)
+
+### Services
+
+CenturyLink Cloud offers different services for which the cost estimation can be found under the individual service page.
+
+Appfog
```
Comment (chrislittle) [inline Control Portal/understang-cost estimates in control.md]: same comment as above

File: Control Portal/understang-cost estimates in control.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+  "contentIsHTML": false
+}}}
+
+
+### Contents
+- [Overview](#overview)
+- [Dashboard](#main-dashboard)
+- [Data Center View](#data-center-view)
+- [Group View](#connectvpn)
+- [Server View](#deployowncloud)
```
Comment (chrislittle) [inline Control Portal/understang-cost estimates in control.md]: #deployowncloud ?

File: Control Portal/understang-cost estimates in control.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+
+### Contents
+- [Overview](#overview)
+- [Dashboard](#main-dashboard)
+- [Data Center View](#data-center-view)
+- [Group View](#connectvpn)
```
Comment (chrislittle) [inline Control Portal/understang-cost estimates in control.md]: #connectvpn ?

File: Control Portal/understang-cost estimates in control.md [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+- [Server View](#deployowncloud)
+- [Network](#network)
+- [Services](#services)
+- [Billing](#billing)
+- [Summary](#summary)
+
+### Overview
+One of the best features of the Control Portal is the cost estimation in every level within the account.  In this knowledge article, we will have a look at what is included in each level of the Control Portal.
+Monthly and hourly estimates are found in the following levels of the Portal:
```
Comment (chrislittle) [inline Control Portal/understang-cost estimates in control.md]: should this be a space or why is it wrapping text?

**Comment by gavin-lai** [comment]
Fixed the issues

**Comment by chrislittle** [comment]
@gavin-lai looks good.  @srveit this can be merged

**Pull request merged**